### PR TITLE
Feat/Add `yml` config

### PR DESCRIPTION
**Pull request by carpawell**
Adopted https://github.com/nspcc-dev/neofs-dev-env/pull/241.

### CODE REVIEW

**Review by acid-ant — DISMISSED** [review]

File: services/s3_gate/docker-compose.yml [hunk 1]
```diff
@@ -17,14 +17,12 @@ services:
...
+      - ./cfg:/etc/frostfs/s3
     stop_signal: SIGKILL
     env_file: [ ".env", ".s3.env", ".int_test.env" ]
+    command: [ "frostfs-s3-gw", "--config", "/etc/frostfs/s3/config.yml" ]
     environment:
-      - S3_GW_WALLET_PATH=/wallet.json
-      - S3_GW_WALLET_PASSPHRASE=s3
       - S3_GW_RPC_ENDPOINT=http://morph-chain.${LOCAL_DOMAIN}:30333
-      - S3_GW_SERVER_0_TLS_KEY_FILE=/tls.key
-      - S3_GW_SERVER_0_TLS_CERT_FILE=/tls.crt
```
Comment (KirillovDenis) [inline services/s3_gate/docker-compose.yml:27]: Why we dropped this here but not added to `config.yml`?

File: services/http_gate/cfg/config.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+logger:
+  level: debug
+  
+rebalance_timer: 5m  # Interval to check nodes health
+
+connect_timeout: 60s  # Timeout to dial node
+
+request_timeout: 300s  # Timeout to check node health during rebalance
```
Comment (KirillovDenis) [inline services/http_gate/cfg/config.yml:8]: Let's set the same as for `connect_timeout`

File: services/http_gate/cfg/config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# The order in which resolvers are used to find an container id by name
+resolve_order:
+  - nns
+
+server:
+  - address: 0.0.0.0:8080
```
Comment (KirillovDenis) [inline services/http_gate/cfg/config.yml]: Why did we changed the port (`80`-> `8080`)?

File: services/rest_gate/cfg/config.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# The IP and port to listen on.
+listen-address: 0.0.0.0:8090
```
Comment (KirillovDenis) [inline services/rest_gate/cfg/config.yml]: It seems here we should use 
```yml
server:
  listen-address: 0.0.0.0:8090
```
and drop the `REST_GW_SERVER_LISTEN_ADDRESS` from `rest_gate/docker-compose.yml`

File: services/http_gate/cfg/config.yml (hunk 2 above)
Comment (alexvanin) [reply services/http_gate/cfg/config.yml:8]: I think @carpawell was just copying ENV params. I propose to create an issue and fix actual values in separate PR.

File: services/s3_gate/docker-compose.yml (hunk 1 above)
Comment (carpawell) [reply services/s3_gate/docker-compose.yml:27]: do not remember exactly. mb mistake. but on the other hand is it expected to work if no [`enabled`](https://github.com/nspcc-dev/neofs-s3-gw/blob/50d85dc7edabe6a753c346c388bf18bf9134cd90/config/config.yaml#L31) config is provided?

File: services/http_gate/cfg/config.yml (hunk 2 above)
Comment (carpawell) [reply services/http_gate/cfg/config.yml:8]: yeah, did not change anything, tried to just make that work. suggest making any value changes in a separate PR

File: services/http_gate/cfg/config.yml (hunk 3 above)
Comment (carpawell) [reply services/http_gate/cfg/config.yml]: fixed

File: services/rest_gate/cfg/config.yml (hunk 4 above)
Comment (carpawell) [reply services/rest_gate/cfg/config.yml]: yeah, inaccurate rebase and bad copy/paste, fixed

File: services/s3_gate/docker-compose.yml (hunk 1 above)
Comment (carpawell) [reply services/s3_gate/docker-compose.yml:27]: added

File: services/http_gate/cfg/config.yml (hunk 2 above)
Comment (KirillovDenis) [reply services/http_gate/cfg/config.yml:8]: https://github.com/TrueCloudLab/frostfs-dev-env/issues/5

**Review by KirillovDenis — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]

**Pull request merged**